x/stake/client/cli/query.go

```diff
@@ -215,6 +215,43 @@ func GetCmdQueryDelegations(storeName string, cdc *codec.Codec) *cobra.Command {
 	return cmd
 }
 
+// GetCmdQueryValidatorDelegations implements the command to query all the
+// delegations to a specific validator.
+func GetCmdQueryValidatorDelegations(queryRoute string, cdc *codec.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "delegations-to [validator-addr]",
```

Comment:
what about `delegations-to-validator [validator-addr]` ?

Reply:
Think that might be a bit too verbose, no?